README.md

```diff
@@ -976,26 +976,28 @@ const state = form.state(o, {
 });
 ```
 
-To implement hidden behavior, pass in an `isHidden` function. You can also
-determine whether a repeating form is disabled from add and remove using
-`isRepeatingFormDisabled`. It's up to you to use this information to render the
-add and remove buttons with the disabled status, however.
+To implement hidden behavior, pass in an `isHidden` function.
 
 To implement readOnly behavior, pass in an `isReadOnly` function.
 
 To implement required behavior, pass in an `isRequired` function. This does not
-only affect the `required` property on fieldAccessor, but also makes the field
-require the field just as if you used the `required` flag in the field
-definition. The `required` flag in the field definition always makes something
-required, no wonder what `isRequired` says.
+only affect the `required` property on the accessor, but also makes the form or
```

Comment:
for required the situation is different. I think you should be clear in the documentation that 'required' is only for fields somehow.